### PR TITLE
Add additional english quotes

### PR DESCRIPTION
**Pull request by pie-shifted**
### Description

Added 2 quotes to 'english.json'.

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -35652,6 +35652,18 @@
     "source": "By The River Piedra I Sat Down And Wept",
     "length": 219,
     "id": 6096
-}
+    },
+    {
+      "text": "It is difficult for me to imagine what \"personal liberty\" is enjoyed by an unemployed hungry person. True freedom can only be where there is no exploitation and oppression of one person by another; where there is not unemployment, and where a person is not living in fear of losing his job, his home and his bread. Only in such a society personal and any other freedom can exist for real and not on paper.",
```
Comment (Bruception) [inline frontend/static/quotes/english.json:35657]: I'm not sure if a quote from Stalin is acceptable. @Miodec

Comment (Bruception) [inline frontend/static/quotes/english.json:35657]: I'm not sure if a quote from Stalin is acceptable. @Miodec

Comment (Miodec) [reply frontend/static/quotes/english.json:35657]: Remove please

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by pie-shifted** [comment]
fixed now -_-

**Comment by Bruception** [comment]
Not sure if we want this in our commit history:

Screen Shot 2022-06-14 at 12 03 52 AM


Please re-open the PR or change the commit message.

**Pull request closed without merging**